src/Service.Host/index.html

```diff
@@ -1,19 +1,19 @@
-﻿<!doctype html>
+﻿<!DOCTYPE html>
 <html>
-<head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Products (local hotload)</title>
-</head>
-<body>
-    <div id="root"></div>
-    <script type="text/javascript">
-        window.APPLICATION_SETTINGS = {
-            authorityUri: 'https://www-sys.linn.co.uk/auth/',
-            appRoot: 'http://localhost:51798',
-            proxyRoot: 'http://app.linn.co.uk'
-        };
-    </script>
-    <script src="/products/maint/build/app.js"></script>
-</body>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>Products (local hotload)</title>
+    </head>
+    <body>
+        <div id="root"></div>
+        <script type="text/javascript">
+            window.APPLICATION_SETTINGS = {
+                authorityUri: 'https://www-sys.linn.co.uk/auth/',
+                appRoot: 'http://localhost:5050',
```

Comment:
Why did you change this. Would only be 5050 if you were running locally in a docker?

Reply:
I've reverted that back on a commit, coming to github soon. It was just the port that the server runs on when you run the service via the command line